components/has/staging/kustomization.yaml

```diff
@@ -3,3 +3,6 @@ kind: Kustomization
 resources:
   - ../base
   - ../base/external-secrets
+
+patches:
+  - ./gitops-org-patch.yaml
```

Comment:
Do we need to have the token patch and target name as it's configure in the production kustomization.yaml file?

Reply:
No, the default external secret resource exists under the base folder, and points to staging.